image/oci.go

```diff
@@ -213,6 +209,18 @@ func (m *manifestOCI1) convertToManifestSchema2() (types.Image, error) {
 	return memoryImageFromManifest(m1), nil
 }
 
+func (m *manifestOCI1) convertToManifestSchema1(ctx context.Context, updateInfo types.ManifestUpdateInformation) (types.Image, error) {
+	m2, err := m.convertToManifestSchema2(ctx, updateInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	return m2.UpdatedImage(ctx, types.ManifestUpdateOptions{
+		ManifestMIMEType: manifest.DockerV2Schema1MediaType,
```

Comment:
(Absolutely non-blocking: `DockerV2Schema1SignedMediaType` is a tiny bit more correct, all schema1 manifests must contain a the signature. OTOH it doesn’t show up in the data anyway…)